### PR TITLE
fix(email): add manage account button in new device login email template

### PR DESCRIPTION
**Pull request by vbudhram**
Fixes https://github.com/mozilla/fxa-auth-server/issues/2197

Adds manage account button in `New sign-in` email. Approved in https://github.com/mozilla/fxa-auth-server/pull/2747

screen shot 2018-12-10 at 11 01 32 am

### CODE REVIEW

File: lib/senders/email.js [hunk 1]
```diff
@@ -911,7 +911,7 @@ module.exports = function (log, config) {
   Mailer.prototype.newDeviceLoginEmail = function (message) {
     log.trace({ op: 'mailer.newDeviceLoginEmail', email: message.email, uid: message.uid })
     var templateName = 'newDeviceLoginEmail'
-    var links = this._generateLinks(this.initiatePasswordChangeUrl, message.email, {}, templateName)
+    var links = this._generateLinks(this.accountSettingsUrl, message.email, {}, templateName)
```
Comment (vladikoff) [inline lib/senders/email.js]: Woo new link!

Comment (vladikoff) [reply lib/senders/email.js]: Why is `initiatePasswordChangeUrl` not here anymore?

File: lib/senders/templates/_versions.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "lowRecoveryCodesEmail": 2,
-  "newDeviceLoginEmail": 1,
+  "newDeviceLoginEmail": 5,
```
Comment (vladikoff) [inline lib/senders/templates/_versions.json]: what was the logic here, v5 vs v2?

File: lib/senders/email.js (hunk 1 above)
Comment (vbudhram) [reply lib/senders/email.js]: `this._generateLinks` first arg is actually the primary callout button link, which in this case we want it to be `accountSettingsUrl`.  The `initiatePasswordChangeUrl` is still created in the links so the user can change password.

File: lib/senders/templates/_versions.json (hunk 2 above)
Comment (vbudhram) [reply lib/senders/templates/_versions.json]: Maybe some scripting strangeness? I'll update.

Comment (shane-tomlinson) [reply lib/senders/templates/_versions.json]: I had that question too.

**Comment by shane-tomlinson** [comment]
When I open the link, `utm_content` is `fxa-password-change` which seems wrong:

> http://127.0.0.1:3030/signin?utm_medium=email&utm_campaign=fx-new-device-signin&utm_content=fx-password-change

**Comment by vbudhram** [comment]
> When I open the link, `utm_content` is `fxa-password-change` which seems wrong:
> 
> > http://127.0.0.1:3030/signin?utm_medium=email&utm_campaign=fx-new-device-signin&utm_content=fx-password-change

@shane-tomlinson This has been updated to be `manage-account` similar to other links.

**Pull request merged**